info/container.go

```diff
@@ -467,6 +467,8 @@ type Usage struct {
 type DerivedStats struct {
 	// Time of generation of these stats.
 	Timestamp time.Time `json:"timestamp"`
+	// Latest instantaneous sample.
+	LatestUsage Usage `json:"latest_usage"`
```

Comment:
I'm okay adding this, do you think it is confusing that we expose the average, 90, max even though they are all the same?

Would it be okay to say to users, if you want instantaneous data go to the /stats endpoint? WDYT?